### ISSUE TITLE
Gracefully handle revoked Device

### ISSUE THREAD

**Issue by thegcat** [pretix/pretixscan-desktop#45]
After a device has been revoked starting the desktop app will yield an error and a stack trace. This should be better handled.

<img width="664" alt="Bildschirmfoto 2019-08-05 um 15 39 15" src="https://user-images.githubusercontent.com/22835/62468821-452c8100-b797-11e9-997c-1a8bdffd9739.png">

**Comment by raphaelm** [comment]
Yes, absolutely! It probably already is handled better during sync, but not during event selection.

**Comment by thegcat** [comment]
That was actually on startup after the key was revoked. Before the app was closed and the key revoked no event was select, so maybe this explains that.

**Comment by raphaelm** [comment]
Fixed in https://github.com/pretix/pretixscan-desktop/commit/a652cca264a276821435423a82c40878ee6e8895
